### PR TITLE
Home

### PR DESCRIPTION
**Pull request by kiranpratyush**
Added Home Page
https://user-images.githubusercontent.com/77445957/160426711-aee6a7df-af27-4b8d-bb36-42bbe6d88eca.mp4

Note:- Some Checks are not passed because of some dependencies are not available in this branch ignore for now

### CODE REVIEW

**Review by techieshubham30 — APPROVED** [review]
Code looks clean and readable, good to go.

File: src/Components/Header/Header.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+.first-nav {
+  display: flex;
+  justify-content: space-evenly;
+  gap: 10px;
+}
+/* .first-nav a {
+  color: var(--color-black);
+} */
```
Comment (MiheerTamkhane) [inline src/Components/Header/Header.css:50]: remove comments if not needed

File: src/Components/HeroContainer/Hero.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```
Comment (MiheerTamkhane) [inline src/Components/HeroContainer/Hero.js:1]: you can also remove react imports if not used!

File: src/Contexts/CartContext.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+
+function useCartContext()
+{
+    const {state,dispatch} = useContext(CartContext)
+    return {state,dispatch}
+}
+function CartContextProvider({children})
+{
+    const [state,dispatch] = useReducer(reducerfn,{data:[],wishList:[]})
```
Comment (MiheerTamkhane) [inline src/Contexts/CartContext.js:31]: I might wrong but giving a function name with '-' hiphen may not be good, camelCase is good practice

File: src/Pages/HomePage.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Header } from '../Components/Header/Header.js';
+import { Main } from '../Components/Main/Main.js';
+import { Footer } from '../Components/Footer/Footer.js';
```
Comment (MiheerTamkhane) [inline src/Pages/HomePage.js:4]: Make a barrel exports for this

**Review by MiheerTamkhane — APPROVED** [review]
Other than that code looks good Good work and nice UI man

File: src/Components/Header/Header.css (hunk 1 above)
Comment (kiranpratyush) [reply src/Components/Header/Header.css:50]: Thank you for pointing out will do

File: src/Components/HeroContainer/Hero.js (hunk 2 above)
Comment (kiranpratyush) [reply src/Components/HeroContainer/Hero.js:1]: Thank you removed

File: src/Contexts/CartContext.js (hunk 3 above)
Comment (kiranpratyush) [reply src/Contexts/CartContext.js:31]: Thank you for the suggestion changed it

File: src/Pages/HomePage.js (hunk 4 above)
Comment (kiranpratyush) [reply src/Pages/HomePage.js:4]: Will do .

**Pull request merged**